code/src/components/AddTodo.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import uniqid from 'uniqid';
+import moment from 'moment';
+
+import todos from 'reducers/todos'
+import {
+  Form, 
+  Input,
+  AddButton,
+} from 'Styles'
+
+const AddTodo = () => {
+  const [inputValue, setInputValue] = useState("");
+
+  const dispatch = useDispatch();
+
+  const onFormSubmit = (event) => {
+    event.preventDefault();
+
+    const newTodo = {
+      id: uniqid(),
+      title: inputValue,
+      isDone: false,
+    }
+
+    dispatch(todos.actions.addItem(newTodo));
+
+    setInputValue("");
+  };
+
+  const date = moment().format('MMMM Do YYYY');
+
+  return (
+      <Form onSubmit={onFormSubmit}>
+        <p>{date}</p>
+          <label>
+          <Input 
+          type="text" 
+          value={inputValue}
+          onChange={(e) => setInputValue(e.target.value)}
+          placeholder="Add new task"
+          />
+          <AddButton type="submit">
```

Comment:
I notice that users could add empty tasks. There're many ways to go about and prevent this. I think the most common way would be to disable the add task button if the input is empty.

Reply:
Will also do this right away!! Thanks!